js/adapt-stateful-session.js

```diff
@@ -79,6 +79,24 @@ define([
 			this.listenTo(Adapt.blocks, 'change:_isComplete', this.onBlockComplete);
 			this.listenTo(Adapt.course, 'change:_isComplete', this.onCompletion);
 			this.listenTo(Adapt, 'assessment:complete', this.onAssessmentComplete);
+			this.listenTo(Adapt, 'app:resetSession', this.onResetSession);
```

Comment:
I'm with @tomgreenfield - the event originates from language picker and presumably is only triggered if the user chooses a different language.. shouldn't the event therefore be `languagePicker::languageChange`?